Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:18-alpine AS base
+WORKDIR /opt/isomercms-backend
+COPY . .
+RUN npm ci
+EXPOSE "8081"
+COPY . /opt/isomercms-backend
```

Comment:
what's this copy for? there was already a copy on line 3

Reply:
removing the one on line 3, is extra